test/integration/guest_can_add_item_to_cart_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class GuestCanAddItemToCartTest < ActionDispatch::IntegrationTest
+
+  test "guest can view item added to cart" do
+    items = create_list(:item, 2)
+
+    visit items_path
+
+    first(:button, 'Add to Cart').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.